SimGeneral/MixingModule/python/caloTruthProducer_cfi.py

```diff
@@ -46,6 +46,33 @@
     simHitCollections = dict(hgc = caloParticles.simHitCollections.hgc + [cms.InputTag('g4SimHits','HFNoseHits')])
 )
 
+from Configuration.Eras.Modifier_run3_ecalclustering_cff import run3_ecalclustering
+run3_ecalclustering.toReplaceWith(caloParticles, cms.PSet(
```

Comment:
Lot's of the content looks like copy-paste from the `caloParticles` definition. It would be much better to override only the necessary parameters.

Can you foresee any need to use the ECAL CaloParticles for phase2 (with HGCal)?